### PR TITLE
add timestamp and expiry functionality to hybridauth_sesstion storage

### PR DESCRIPTION
**Pull request by eugene-o-z**

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
         class:  Azine\HybridAuthBundle\Services\AzineHybridAuth
         arguments:
             router:               "@router"
-            tokenStorage:      "@security.token_storage"
+            token:                "@=service('service_container').get('security.context').getToken().getUser()"
```
Comment (azine) [inline Resources/config/services.yml]: isn't the "security.context" deprecated from Symfony 2.7 onwards?

File: Services/AzineHybridAuth.php [hunk 2]
```diff
@@ -53,23 +58,23 @@ class AzineHybridAuth {
...
+	 * @param UserInterface $user
 	 * @param TokenStorageInterface $tokenStorage
 	 * @param ObjectManager $manager
 	 * @param array $config
 	 * @param bool $storeForUser
 	 * @param $storeAsCookie
+	 * @param $expiresInDays
 	 */
-	public function __construct(UrlGeneratorInterface $router, TokenStorageInterface $tokenStorage, ObjectManager $manager, $config, $storeForUser, $storeAsCookie){
+	public function __construct(UrlGeneratorInterface $router, UserInterface $user, ObjectManager $manager, $config, $storeForUser, $storeAsCookie, $expiresInDays){
```
Comment (azine) [inline Services/AzineHybridAuth.php]: Why does this need to be changed? If possible, the public function signature should stay the same, so users don't have problems with BC breaks.

File: Resources/config/services.yml [hunk 3]
```diff
@@ -18,6 +18,7 @@ services:
                 debug_file:       "%azine_hybrid_auth_debug_file%"
             store_for_user:   "%azine_hybrid_auth_store_for_user%"
             store_as_cookie:  "%azine_hybrid_auth_store_as_cookie%"
+            expires_in_days:  "%azine_hybrid_auth_expires_in_days%"
```
Comment (azine) [inline Resources/config/services.yml]: this specifies that an int is passed as parameter to the constructor of AzineHybridAuth. But the current constructor does not take an int as argument.

File: Services/AzineHybridAuth.php [hunk 4]
```diff
@@ -39,6 +39,11 @@ class AzineHybridAuth {
 	private $storeAsCookie;
 
 	/**
+	 * @var int
+	 */
+	private $expiresInDays;
```
Comment (azine) [inline Services/AzineHybridAuth.php]: $expiresInDays is never initialized. I guess it should be part of the __constructor.

Comment (eugene-o-z) [inline Services/AzineHybridAuth.php]: Sorry, just erased this incidentally in previous commit.

**Pull request merged**